examples/reverse_words/__init__.py

```diff
@@ -89,6 +90,22 @@ def get_dim(self, name):
         return super(Transition, self).get_dim(name)
 
 
+def lower(s):
```

Comment:
Let's follow the pattern of having underscores in front of such dumb helpers similarly to the way you do in `sqrt.py`.